iXBRLViewerPlugin/viewer/src/less/common.less

```diff
@@ -2,12 +2,12 @@
 
 /* Styles common to both inspector and highlighting used within iframe */
 .linked-highlight-text {
-  outline: dashed 2px @linked-fact;
-  outline-offset: 1px;
+  outline: dashed 0.125em @linked-fact;
```

Comment:
Nice.  I wish I'd thought of this when I previously tried and failed to fix the issue with massively scaled documents.